src/test/testing_assert.cpp

```diff
@@ -106,8 +100,9 @@ std::string _matrix_to_string(const Matrix& matrix, const std::vector<std::pair<
   return stream.str();
 }
 
-template <typename T, typename std::enable_if<std::is_floating_point<T>::value>::type* = nullptr>
-bool near(T left_val, T right_val, opossum::FloatComparisonMode float_comparison_mode) {
+template <typename T>
+bool almost_equals(T left_val, T right_val, opossum::FloatComparisonMode float_comparison_mode) {
+  static_assert(std::is_floating_point_v<T>, "Values must be have floating point type.");
```

Comment:
Values must ~be~ have floating point type.

Reply:
done